index.js

```diff
@@ -2,45 +2,57 @@ var app = {
   version: 1,
   currentQuestionNumber: 0,
   jsonFile: "questions.json",
-  area: $('.gameArea'),
-  board: $('.gameBoard'),
-  buttons: $(".btnHolder"),
+  area: document.querySelector('.gameArea'),
+  board: document.querySelector('.gameBoard'),
   // Utility functions
   jsonLoaded: function (data) {
     console.clear()
     app.questions = data;
     app.showTitleThenQuestion(0);
   },
   disablePointsButtons: function () {
-    app.buttons.find("#awardTeam1").prop("disabled",true);
-    app.buttons.find("#awardTeam2").prop("disabled",true);
+    document.querySelector("#awardTeam1").disabled = true;
+    document.querySelector("#awardTeam2").disabled = true;
   },
   enablePointsButtons: function () {
-    app.buttons.find("#awardTeam1").prop("disabled",false);
-    app.buttons.find("#awardTeam2").prop("disabled",false);
+    document.querySelector("#awardTeam1").disabled = false;
+    document.querySelector("#awardTeam2").disabled = false;
   },
   resetBoard: function () {
-    var boardScore = app.board.find("#boardScore");
-    boardScore.html(0)
-    var col1 = app.board.find(".col1");
-    var col2 = app.board.find(".col2");
-    col1.empty();
-    col2.empty();
+    var boardScore = app.board.querySelector("#boardScore");
+    boardScore.innerHTML = 0;
+    var col1 = app.board.querySelector(".col1");
+    var col2 = app.board.querySelector(".col2");
+    //empty out the two columns
+    while(col1.firstChild)
```

Comment:
This method is a little slow. You might want to just set innerHTML to empty string.